### ISSUE TITLE
False positive for `t.Errorf`

### ISSUE THREAD

**Issue by SVilgelm** [alingse/asasalint#17]
I tried to enable the linter with golangci-lint and it fails on the `t.Errorf` function, it is same as `Printf`, should be ignored
```go
cond_expr_test.go:472:44: pass []any as any to func t.Errorf func(format string, args ...any) (asasalint)
                                        t.Errorf("arguments don't match: %v", arguments)
                                                                              ^
```

**Comment by alingse** [comment]
fixed
